src/components/flow/routers/classify/__snapshots__/ClassifyRouterForm.test.tsx.snap

```diff
@@ -2379,644 +2379,6 @@ exports[`ClassifyRouterForm > should render 1`] = `
 </body>
 `;
 
-exports[`ClassifyRouterForm > should render with an incorrect action 1`] = `
```

Comment:
Why was this snapshot removed if none of the modified files are related to it?
Was the CI breaking before the PR changes due to this snapshot?
It's strange that the current CI is passing, indicating that this snapshot really should be removed 😕 
Going to approve due to that, but we should keep an eye in this snapshot later if it comes and goes between PRs.